crates/router/src/connector/trustpay.rs

```diff
@@ -148,7 +152,11 @@ impl ConnectorCommon for Trustpay {
             }
             Err(error_msg) => {
                 logger::error!(deserialization_error =? error_msg);
-                utils::handle_json_response_deserialization_failure(res, "trustpay".to_owned())
+                utils::handle_json_response_deserialization_failure(
```

Comment:
We need not to pass this to the method, Instead keep the method as it is and log before calling the method.

Reply:
As we convert connector response from bytes to string, within the method. Is it okay to log it within the method itself. Otherwise we will have to repeat the conversion, for each of the connector.